### DISCUSSION TITLE
Can I use the layout for a custom page not related to CRUD?

### DISCUSSION THREAD

**Discussion by NuktukDev** [Laravel-Backpack/community-forum#956] [Q&A (Help)]
Put simply, if I wanted a standalone page, but I still wanted to use the layout that the crud system is using, in my case it would be the tabler theme, how would I go about doing that?

**Comment by karandatwani92** [comment]
Yes, you can use it. You just need `@extends(backpack_view('blank'))`. Here is a quick example in the demo:
Page: https://demo.test/admin/pet-shop/about
Blade View: https://github.com/Laravel-Backpack/demo/blob/main/resources/views/admin/petshop_about.blade.php
Route:
```php
Route::get('about', function () {
   return view('admin.petshop_about');
});
```
